clients/render/components/Fields/FileInput.js

```diff
@@ -0,0 +1,197 @@
+import {CalderaFormsFieldGroup, Fragment} from "../CalderaFormsFieldGroup";
+import {CalderaFormsFieldPropType} from "../CalderaFormsFieldRender";
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import Dropzone from 'react-dropzone';
+
+
+export const FileInput = ( props )  => {
+
+    const { shouldDisable,accept,field,describedById,onChange,style,className,multiple,multiUploadText,inputProps, text, previewStyle} = props;
+    const {
+      outterIdAttr,
+      fieldId,
+      fieldLabel,
+      fieldCaption,
+      required,
+      fieldPlaceHolder,
+      fieldDefault,
+      fieldIdAttr,
+      isInvalid
+    } = field;
+
+    let {
+      fieldValue
+		} = field;
+
+  if( Array.isArray(fieldValue) === false ) {
+    fieldValue = [];
+  }
+
+  const onDrop = (accepted) => {
+
+    accepted.forEach( file => {
+      fieldValue.push(file);
+    })
+
+    onChange( fieldValue );
+
+  };
+
+  const removeFile = (e, file) => {
+
+    const index = fieldValue.indexOf(file);
+    fieldValue.splice(index, 1);
+
+    onChange( fieldValue );
+
+  }
+
+  let ulExpanded;
```

Comment:
@New0 This naming is a little confusing. `isListExpanded` is more obvious.

Also, make it one line - `const isListExpanded = fieldValue.length > 0;`